addons/html_builder/static/src/builder/snippets_menu.js

```diff
@@ -230,6 +221,25 @@ export class SnippetsMenu extends Component {
         );
         return result;
     }
+    /**
+     * If the element holds a translation, saves it. Otherwise, fallback to the
+     * standard saving but with the lang kept.
+     *
+     */
+    async saveTranslationElement(el) {
+        if (el.dataset["oeTranslationSourceSha"]) {
+            const translations = {};
+            translations[this.websiteService.currentWebsite.metadata.lang] = {
+                [el.dataset["oeTranslationSourceSha"]]: el.innerHTML,
+            };
+            return this.orm.call(el.dataset["oeModel"], "web_update_field_translations", [
+                [Number(el.dataset["oeId"])],
+                el.dataset["oeField"],
+                translations,
+            ]);
+        }
+        return this.saveView(el);
```

Comment:
For the moment, this is only useful for image fields. One big limitation for the moment is that in transle mode, although the option previews work correctly, it is currently not possible to save something else than text change (except an image field change).

Reply:
It s the same thing in master ?